treeherder/extract/extract_alerts.json

```diff
@@ -93,7 +93,8 @@
     "logs": [
       {
         "log_type": "logger",
-        "name": "extract_alerts"
+        "name": "extract_alerts",
+        "min_level": "warning"
```

Comment:
PaperTrail does not have the extract-to-bigquery logs:  It seems only the WARNING log lines are emitted.  This raises the logging level of the extractor so that they get through to PaperTrail.

Maybe there is a command line switch to ensure the INFO log lines are emittted?  Right now the command is:

    newrelic-admin run-program ./manage.py extract_jobs --force